### ISSUE TITLE
When I use RIDE to open __init__.html, it always report "invalid syntax" error 

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#752]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=717) by sean.li... on 28 Feb 2011*

When I use RIDE to open __init__.html, it always report following error:

"Invalid syntax in file 'D:
	\TA\TestCase\DATest\SS\__init__.html' in table 'Setting': Non-existing setting 'Suite Setup'.
Invalid syntax in file 'D:
	\TA\TestCase\DATest\SS\__init__.html' in table 'Setting': Non-existing setting 'Suite Teardown'.
Invalid syntax in file 'D:
	\TA\TestCase\DATest\SS\__init__.html' in table 'Setting': Non-existing setting 'Force Tags'."

Robot Framework 2.5.6 (Python 2.6.6 on win32)
RIDE 0.32.1

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=717#c1) by sean.li... on 28 Feb 2011*


It is from IPA internal

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=717#c2) by @mkorpela on 4 Mar 2011*


RIDE can open only file suites or directory suites.

Try open \TA\TestCase\DATest\SS\ directory.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=717#c4) by sean.li... on 15 Mar 2011*


Open a full directory case may cause other problems, there are so many cases in that directory. If we could edit via __init__.html directly, that would be very nice.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=717#c5) by @yanne on 17 Mar 2011*


This actually requires a small change to Robot Framework itself and can be implemented only after RF code comes bundled with RIDE.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=717#c6) by @yanne on 17 Mar 2011*


<a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=749" title="ride do not support to open __init__.html directly"> Issue 749 </a> has been merged into this issue.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=717#c9) by @yanne on 12 Aug 2011*


This issue was updated by <a href="https://code.google.com/p/robotframework-ride/source/detail?r=c267d2a27350">revision c267d2a27350</a>.
